frontend/tests/components/input-pattern-field.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { render } from '@testing-library/react';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+
+import { InputPatternField } from '~/components/input-pattern-field';
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key,
+  }),
+}));
+
+describe('InputPatternField', () => {
+  afterEach(() => {
+    vi.clearAllMocks();
+    vi.resetModules();
+  });
```

Comment:
This is not requird because of <https://github.com/DTS-STN/future-sir/blob/main/frontend/tests/setup.ts#L8>